web/voice_chat/signaling.py

```
@@ -0,0 +1,78 @@
+        await self.channel_layer.group_send(
+            self.room_group_name,
+            {
+                'type': 'user_left',
+                'user_id': self.scope['user'].id,
+                'username': self.scope['user'].username
+            }
+        )
+
+    async def receive(self, text_data):
+        data = json.loads(text_data)
+        message_type = data['type']
```

Ruff (0.8.2)

Local variable `message_type` is assigned to but never used

Remove assignment to unused variable `message_type`

(F841)